LoRaEngine/LoraKeysManagerFacade/IoTCentralImp/IoTCentralDeviceRegistryManager.cs

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade.IoTCentralImp
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Threading.Tasks;
+    using LoraKeysManagerFacade.IoTCentralImp.Definitions;
+    using Newtonsoft.Json.Linq;
+
+    internal class IoTCentralDeviceRegistryManager : IDeviceRegistryManager
+    {
+        private const string API_VERSION = "api-version=1.1-preview";
```

Comment:
Looking at the code, you don't have any other way to override this constant. It would be nice to check if an environment variable exists for this one and override it. Otherwise, if the API is changing , go for a release and API are compatible, you'd have to fully rebuild all. 
Note: I would most likely not leave this comment if it would be a non preview API